src/test/java/software/amazon/awssdk/crt/test/HttpRequestResponseTest.java

```diff
@@ -357,5 +357,4 @@ public void testHttpUpload() throws Exception {
         Assert.assertNotNull("Response Body did not contain \"data\" JSON key:\n" + body, echoedBody);
         Assert.assertEquals(bodyToSend, echoedBody);
     }
-
 }
```

Comment:
Add a unit test for creating a request without activating it. make sure things clean up properly